packages/components/src/design-tokens.ts

```diff
@@ -174,6 +173,12 @@ export {
 
 const { create } = DesignToken;
 
+/**
+ * The default disabled opacity.
+ */
+export const disabledOpacity =
```

Comment:
Actually as the design token are shared, you should only call `withDefault` on the imported token:

```ts
import { disabledOpacity } from '@microsoft/fast-components';
export { disabledOpacity } from '@microsoft/fast-components';

disabledOpacity.withDefault(0.3);
```